lib/coinbase/client/api/external_addresses_api.rb

```diff
@@ -94,6 +94,91 @@ def get_external_address_balance_with_http_info(network_id, address_id, asset_id
+    # @return [AddressHistoricalBalanceList]
+    def list_address_historical_balance(network_id, address_id, asset_id, opts = {})
+      data, _status_code, _headers = list_address_historical_balance_with_http_info(network_id, address_id, asset_id, opts)
+      data
+    end
+
+    # Get address balance history for asset
+    # List the historical balance of an asset in a specific address.
+    # @param network_id [String] The ID of the blockchain network
+    # @param address_id [String] The ID of the address to fetch the historical balance for.
+    # @param asset_id [String] The symbol of the asset to fetch the historical balance for.
+    # @param [Hash] opts the optional parameters
+    # @option opts [Integer] :limit A limit on the number of objects to be returned. Limit can range between 1 and 100, and the default is 10.
+    # @option opts [String] :page A cursor for pagination across multiple pages of results. Don&#39;t include this parameter on the first call. Use the next_page value returned in a previous response to request subsequent results.
+    # @return [Array<(AddressHistoricalBalanceList, Integer, Hash)>] AddressHistoricalBalanceList data, response status code and response headers
+    def list_address_historical_balance_with_http_info(network_id, address_id, asset_id, opts = {})
+      if @api_client.config.debugging
+        @api_client.config.logger.debug 'Calling API: ExternalAddressesApi.list_address_historical_balance ...'
+      end
+      # verify the required parameter 'network_id' is set
+      if @api_client.config.client_side_validation && network_id.nil?
+        fail ArgumentError, "Missing the required parameter 'network_id' when calling ExternalAddressesApi.list_address_historical_balance"
+      end
+      # verify the required parameter 'address_id' is set
+      if @api_client.config.client_side_validation && address_id.nil?
+        fail ArgumentError, "Missing the required parameter 'address_id' when calling ExternalAddressesApi.list_address_historical_balance"
+      end
+      # verify the required parameter 'asset_id' is set
+      if @api_client.config.client_side_validation && asset_id.nil?
+        fail ArgumentError, "Missing the required parameter 'asset_id' when calling ExternalAddressesApi.list_address_historical_balance"
+      end
+      if @api_client.config.client_side_validation && !opts[:'page'].nil? && opts[:'page'].to_s.length > 5000
+        fail ArgumentError, 'invalid value for "opts[:"page"]" when calling ExternalAddressesApi.list_address_historical_balance, the character length must be smaller than or equal to 5000.'
+      end
+
+      # resource path
+      local_var_path = '/v1/networks/{network_id}/addresses/{address_id}/balance_history/{asset_id}'.sub('{' + 'network_id' + '}', CGI.escape(network_id.to_s)).sub('{' + 'address_id' + '}', CGI.escape(address_id.to_s)).sub('{' + 'asset_id' + '}', CGI.escape(asset_id.to_s))
```

Comment:
this is not for staking balance but general account balance, I assume we are adding the support for checking general account balance as well, but did not see [the staking balance equivalent ](https://github.cbhq.net/c3/cdp-service/blob/4b2ae9dabe6de4c494c1fdbd072bffe0deb8a02f/openapi/cdp-service.yml#L1794)
`/v1/networks/{network_id}/addresses/{address_id}/stake/balances`

Reply:
This is automated